src/djcytoscape/models.py

```diff
@@ -119,7 +142,7 @@ class CytoElement(models.Model):
                              help_text="if present, will be used as node label instead of id")
     min_len = models.IntegerField(default=1,
                                   help_text="number of ranks to keep between the source and target of the edge")
-    href = models.URLField(blank=True, null=True)
+    href = models.CharField(blank=True, null=True, max_length=50, validators=[validate_url])
```

Comment:
maybe "validate_local_url"? for greater clarity? if I see validate url I'm thinking http...etc